scene/gui/file_dialog.cpp

```diff
@@ -1111,7 +1111,7 @@ void FileDialog::_update_option_controls() {
 	options_dirty = false;
 
 	while (grid_options->get_child_count(false) > 0) {
-		Node *child = grid_options->get_child(0);
+		Node *child = grid_options->get_child(0, false);
```

Comment:
What is the `false` for? `grid_options` does not have internal children from what I see, so it doesn't matter. Same in EditorFileDialog.

Reply:
It should match above at least

Reply:
Yeah the comment is meant in general. I found it convenient to put here.

Reply:
There's no internal children, so it will work the same with or without it. So it can be safely removed (or kept as is in case some internal children are added later).

Reply:
`grid_options` are already under an internal child, so adding internal nodes here does not make sense.
For FileDialog it can be left as is, because it's unrelated to this PR, but you can remove the `false` from EditorFileDialog calls.

Reply:
Removed `false` from both.